charts/common/templates/_image.yaml

```
@@ -0,0 +1,17 @@
+{{/*
+Creates a Docker image string.
+
+Usage:
+{{ include "common.image" (dict "image" .Values.image "chart" .Chart) }}
+
+Params:
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 2 but found 0

(indentation)